OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,201 @@
-     `DC_USERHOME`, `DC_REGISTRY_SOA`, `DC_REGISTRY_DB`    
-
-SOA/BPM/OSB domain images
--------------------------
-1. You must have the install binaries downloaded from the
-   [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/soasuite/downloads/index.html) site before proceeding. 
-2. From the current directory run these commands. You will 
-   be prompted with all the information. Carefully review 
-   it and Confirm to proceed. 
-
-       # Use BUILD_OPTS to add extra arguments to the docker build command
-       sh buildDockerImage.sh -v 12.2.1.2
+SOA on Docker
+=============
+
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for SOA 12.2.1.2.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.2.0.
+
+At the end of this configuration there will be 3 containers running : 1) DB Container 2) WLS Admin Server Container 3) WLS Managed Server (SOA Server) Container.
+The containers will be connected using a Docker User Defined network 
+
+
+## Pre-Requisite
+
+
+## Create a User Defined network
+
+
+In this configuration creation of a user defined network will enable the communication between the containers just using container names. User defined network option was preferred over the container linking option as the latter is now deprecated.
+For this setup we will use a user defined network using bridge driver.
+
+Create a user defined network using the bridge driver by executing the following command :
+
+    $ docker network create -d bridge <some name>
+   
+Sample command:
+
+    $ docker network create -d bridge SOANet
+
+
+# Mount a host directory as a data volume
+
+Data volumes are designed to persist data, independent of the container’s lifecycle. The default location of the volume in container is under `/var/lib/docker/volumes`. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume. The volume will be used to store Database datafiles and Weblogic server domain files.This volume will be created on the host at `/scratch/DockerVolume/SOAVolume/`. Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
+
+    $ chmod -R 777 `/scratch/DockerVolume/SOAVolume`
+
+**Database**
+
+You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
+
+Please refer README.md under docker/OracleDatabase for details on how to build Oracle Database image.
```

Comment:
Please create a proper Markdown link from the [`README.md`](https://github.com/oracle/docker-images/blob/master/OracleDatabase/README.md) text to the actual file on GitHub, as I've done in this comment.

The syntax is: `[README.md](https://github.com/oracle/docker-images/blob/master/OracleDatabase/README.md)`